lib/autocomplete-paths.js

```diff
@@ -90,16 +93,23 @@ export default {
       }
     }
     this._statusBarInterval = setInterval(() => {
-      const fileCount = this._provider.fileCount;
-      this._statusBarElement.innerHTML = `Rebuilding paths cache... ${fileCount} files`;
+      if (!this._provider) {
+        return
+      }
+      const fileCount = this._provider.fileCount
+      if (fileCount > 0) {
+        this._statusBarElement.innerHTML = `Rebuilding paths cache... ${fileCount} files`
+      }
```

Comment:
I added a provider check in #233 
```suggestion
      const fileCount = this._provider.fileCount;
      if (fileCount > 0) {
        this._statusBarElement.innerHTML = `Rebuilding paths cache... ${fileCount} files`;
      }
```